src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -504,6 +504,7 @@ private function addPropertyAccessSection(ArrayNodeDefinition $rootNode)
                     ->children()
                         ->booleanNode('magic_call')->defaultFalse()->end()
                         ->booleanNode('throw_exception_on_invalid_index')->defaultFalse()->end()
+                        ->scalarNode('cache')->end()
```

Comment:
IMHO it would be fine provide more information, like `->info('Service name used for cache')->example('property_accessor.cache.apc')`